hack/postgres-passwords-migration.sh

```diff
@@ -0,0 +1,30 @@
+#! /bin/bash
+# This script must be run to migrate all database passwords
+# before upgrading the synopsys-operator
+
+function usage(){
+  echo -e "usage:        ${0} [kubectl/oc] [NAMESPACE1] [NAMESPACE2] [...]\n"
+  echo -e "kubectl/oc:   Type which client you would like to use\n"
+  echo -e "NAMESPACE:    The blackduck namespace to migrate the secret from\n"
+}
+
+# check for number of arguments
+[ "$#" -lt 2 ] && echo -e "ERROR: At least one NAMESPACE must be provided.\n" && usage && exit
+
+# get the client; kubectl or oc
+client=$1
+echo -e "Using $client as the client, please make sure you have it on the machine you're running this script on\n"
+# move the argument pointer past the first item to process namespaces
+shift
+
+# Get ADMIN_PASSWORD, USER_PASSWORD, POSTGRES_PASSWORD from synopsys-operator blackduck-secret
+ADMIN_PASSWORD=$($client get secret -n synopsys-operator blackduck-secret -o jsonpath='{.data.ADMIN_PASSWORD}')
+# USER_PASSWORD
+USER_PASSWORD=$($client get secret -n synopsys-operator blackduck-secret -o jsonpath='{.data.USER_PASSWORD}')
+# POSTGRES_PASSWORD
+POSTGRES_PASSWORD=$($client get secret -n synopsys-operator blackduck-secret -o jsonpath='{.data.POSTGRES_PASSWORD}')
+
+for namespace
+do
```

Comment:
Probably better to do a check to ensure the namespace exists and print a nice error message before trying to patch and letting the client print out the failed patch message (which might be confusing)

Reply:
Not really checking for namespace, but if crd `namespace` does not exist, the script will throw from same error as kubectl: `Error from server (NotFound): blackducks.synopsys.com "invalid_namespace" not found`